client/task-list/welcome-modal/illustrations/inbox.js

```diff
@@ -0,0 +1,135 @@
+/**
```

Comment:
the illustrations are tucked away in the welcome-modal sub directory for now, but its trivial to move them later if we want to implement coloring or reuse them. I felt it would be better to wait till those requirements were clear before doing that.